Assets/Prefabs/ObstacleController.prefab

```diff
@@ -44,7 +44,6 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   obstacleTypes:
-  - {fileID: 6654553859295572524, guid: cb9ba66891e0db045a172cce169e29cf, type: 3}
```

Comment:
This was 
<img width="72" alt="image" src="https://user-images.githubusercontent.com/7551576/161402780-1134907e-1888-495d-8081-aba20b88fb4b.png">
which has no behaviour